routes/root.js

```diff
@@ -193,15 +193,52 @@ route.get('/claims/add', auth.ensureLoggedInGithub, (req, res) => {
   })
 })
 
-route.get('/claims/:id', auth.adminOnly, (req, res) => {
-  du.getClaimById(req.params.id)
-    .then(claim => {
-      if (!claim) throw new Error('No claim found')
-      res.render('pages/claims/id', { claim })
-    })
-    .catch(err => {
-      res.send('Error fetching claim id = ' + escapeHtml(req.params.id))
+route.get('/claims/:id', auth.adminOnly, async (req, res) => {
+  try {
+    const claimInfo = await du.getClaimById(req.params.id)
+    if (!claimInfo) {
+      throw new Error('Error finding claim')
+    }
+
+    const prSlug = new URL(claimInfo.pullUrl).pathname.replace(/\/+$/, '')
+    let response, conflicts
+
+    /**
+     * the database has internal checks for duplicate items in the pull request column
```

Comment:
does not cover all the cases.

Reply:
could you please mention which type of case is not handled